.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "exllama"]
 	path = exllama
-	url = https://github.com/turboderp/exllama.git
+	url = https://github.com/technillogue/exllama.git
```

Comment:
what does forking exllama get us here?

Reply:
read further and from context clues it seems the answer is "lora". Did I get that right? :D

Reply:
it removes exllama_ext directory and replaces it with a compiled exllama_ext.so, and changes cuda_ext to disable runtime compilation and just import it 

also disables a spurious assert

https://github.com/technillogue/exllama/compare/master...no-jit-py311